### DISCUSSION TITLE
How do you check whether a run was part of a backfill?

### DISCUSSION THREAD

**Discussion by sryza** [dagster-io/dagster#16189] [Q&A] [answered]
Sourced from slack: https://dagster.slack.com/archives/C01U954MEER/p1693320482816869.

Hi. Is there an option to check whether an asset run was triggered as part of a backfill?

**Comment by sryza** [comment, marked as answer]
Runs that were triggered as part of a backfill will be assigned a backfill tag that includes the backfill ID.  Here's an example of one displayed on the Runs page:

<img width="629" alt="image" src="https://github.com/dagster-io/dagster/assets/654855/81ead9bb-22c6-4064-a172-979a6dfb9a0c">


If you have a `DagsterRun` object that you've fetched from the instance, you can inspect its `tags` attribute.  You can also inspect run tags using the `DagsterInstance.get_run_tags` method.

**Reply by nrsmac** [reply]
hi @sryza -- would you mind providing an example of accessing this tag from an AssetExecutionContext? Can't seem to figure it out from the API Reference.

**Reply by sryza** [reply]
`context.run.tags` should get it for you

**Reply by MauroCE** [reply]
In particular `context.run.tags['dagster/backfill']`
